packages/react-i18n/src/i18n.ts

```diff
@@ -231,6 +254,47 @@ export class I18n {
     return this.formatNumber(amount, {as: 'currency', ...options});
   }
 
+  formatCurrencyExplicit(
```

Comment:
Extra methods feels very awkward compared to an option for the existing `formatCurrency` method

Reply:
I'm happy to change this, but could use a bit of direction re: what you'd like to see.
Maybe something like this?

Extend `NumberFormatOptions`:
```
export interface NumberFormatOptions extends Intl.NumberFormatOptions {
  as?: 'number' | 'currency' | 'percent';
  precision?: number;
  verbosity?: 'short' | 'explicit';
}
```
which results in calls like:
```
formatCurrency(1.25, {currency: 'USD', verbosity: 'short') ==> '$1.25'
```
and
```
formatCurrency(1.25, {currency: 'USD', verbosity: 'explicit') ==> '$1.25 USD'
```

Reply:
I've changed the public API to
```
formatCurrency(1.25, {currency: 'CAD', form: 'short'}) ==> '$1.25'
```
and 
```
formatCurrency(1.25, {currency: 'CAD', form: 'explicit'}) ==> '$1.25 CAD'
```
The legacy version
```
formatCurrency(1.25, {currency: 'CAD'}) ==> '$1.25' or 'CA$1.25' depending on browser/locale
```
remains for backward compatibility with old code.